packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -227,6 +229,23 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
     }
   };
 
+  /**
+   * It's needed to control after state changes because self controlled dialogs
+   */
+  componentDidUpdate() {
```

Comment:
Similar should be for `didMount` as `Dialog` can be opened by default

Reply:
Done